ttnn/cpp/ttnn/operations/conv/conv2d/device/conv2d_op_width_sharded_program_factory.cpp

```diff
@@ -307,9 +315,9 @@ operation::ProgramWithCallbacks multi_core_optimized_conv_width_sharded_v2_impl(
     uint32_t weight_num_subblocks = weight_block_w_ntiles / out_subblock_w_ntiles;
     uint32_t weight_block_h_ntiles = act_block_w_ntiles;
     uint32_t weight_block_num_tiles = weight_block_w_ntiles * weight_block_h_ntiles;
-    uint32_t weight_block_in_channels_ntiles = input_channels_padded/(32*total_num_cores*per_core_num_blocks_act_w);
-    TT_FATAL(input_channels_padded>=(TILE_HEIGHT*total_num_cores), "Error");
-    TT_FATAL(input_channels_padded%(TILE_HEIGHT*total_num_cores)==0, "Error");
+    uint32_t weight_block_in_channels_ntiles = input_channels_padded/(32 * input_num_cores * per_core_num_blocks_act_w);
+    TT_FATAL(input_channels_padded>=(TILE_HEIGHT * input_num_cores), "Error");
+    TT_FATAL(input_channels_padded%(TILE_HEIGHT * input_num_cores)==0, "Error");
```

Comment:
Can you also update the error message to something meaningful?

Reply:
Done.